proto/process/agent.proto

```diff
@@ -115,6 +115,15 @@ message CollectorReqStatus {
 	string hostName = 2;
 }
 
+message CollectorPod {
+	string hostName = 1;
+	repeated Pod pods = 2;
+	int32 groupId = 3;
+	int32 groupSize = 4;
+	// Post-resolved fields
+	int32 hostId = 5;
```

Comment:
I don't think this should be `hostId`.

Let's make it a `Host` object:

```
// Post-resolved fields
Host host = 5;
```

Reply:
Ah yes, I probably copy pasted this from the RT types where these are `int32`